src/transformers/utils/generic.py

```diff
@@ -959,7 +938,7 @@ def wrapper(self, *args, **kwargs):
         return_dict_passed = kwargs.pop("return_dict", return_dict)
         if return_dict_passed is not None:
             return_dict = return_dict_passed
-        output = func(self, *args, **kwargs)
+        output = func(self, *args, **kwargs, return_dict=True)
```

Comment:
This way, it's going to work in case `**kwargs: [TransformersKwargs]` are properly propagated from the top module up to each wrapped module. 

Just for the context, previously, we were recursively setting the module attribute `_is_top_module` to avoid passing `**kwargs` everywhere.